challenge_2/python/OkraArmband/src/unique_ele.py

```diff
@@ -0,0 +1,20 @@
+from collections import defaultdict
+
+def unique(lst):
+    # Super duper lazy way of solving problem... o(n^2) method.
+    unique_lst = [itm for itm in lst if lst.count(itm) == 1]
+    return unique_lst
+
+def unique_v2(lst):
```

Comment:
Try to refrain from using ambiguous function names. Instead, make the name of the function accurate and descriptive of what it is about. For example, `def findUniqueChar():`